lib/gpu_memory_service/integrations/vllm/model_loader.py

```
@@ -346,8 +363,112 @@ def _load_write_mode(
+    their original storage is now free-cached in torch's default pool.
+    ``torch.cuda.empty_cache`` is patched to a no-op while GMS mappings are
+    live; the accelerator entrypoint bypasses that patch and is safe here.
+    """
+    try:
+        gc.collect()
+        accelerator = getattr(torch, "accelerator", None)
+        if accelerator is not None:
+            accelerator.empty_cache()
+        else:
+            torch.cuda.empty_cache()
+    except Exception as e:  # pragma: no cover - best effort reclaim
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)